irc_im.c

```diff
@@ -969,6 +969,31 @@ static gboolean bee_irc_channel_chat_invite( irc_channel_t *ic, irc_user_t *iu )
 	return TRUE;
 }
 
+static void bee_irc_channel_chat_kick( irc_channel_t *ic, irc_user_t *iu, const char *msg )
+{
+	struct groupchat *c = ic->data;
+	bee_user_t *bu = iu->bu;
+	
+	if( bu == NULL )
+		return;
+	
+	if( c && ( iu->bu->ic != c->ic ) )
+	{
+		irc_send_num( ic->irc, 482, "%s :Can't mix different IM networks in one groupchat", ic->name );
```

Comment:
This error message doesn't make sense for kicks. Also the numeric ERR_USERNOTINCHANNEL might make more sense in this particular case.

Is there no easy/efficient way to check if a nick is in a channel here? Not sure if it's okay to leave that to the protocol